core/src/i18n/en.yaml

```diff
@@ -164,6 +170,29 @@ daemon-sets:
     title: Tolerations
     toleration-seconds: Seconds
 deployments:
+  create-modal:
+    advanced:
```

Comment:
If the following strings are headers, consider capitalization - see [English: General Rule for Capitalization (Sentence Style and Title Case) on the User Interface](https://help.sap.com/viewer/DRAFT/e33c591ae4494a659a3f5f983c9d1161/PROD/en-US/79ce282052d14cc38e6a203a1fc277c4.html).